modules/auxiliary/scanner/http/owa_login.rb

```diff
@@ -79,7 +79,7 @@ def initialize
     register_options(
       [
         OptInt.new('RPORT', [ true, "The target port", 443]),
-        OptAddress.new('RHOST', [ true, "The target address", true]),
+        OptAddress.new('RHOST', [ true, "The target address", '']),
```

Comment:
@wvu-r7 This module works flawlessly especially in internal pentest where the Exchange API could be interacted with directly. I agree with @FireFart. It should be landed first and we can work on the code clean-up in a separate PR.